jsons/UnitPromotions.json

```diff
@@ -546,7 +546,7 @@
 	},
 	{
 		"name": "Quick Study", // only for Keshik and subsequent upgrades
-		"uniques": ["[50]% Bonus XP gain"]
+		"uniques": ["[50]% XP gained from combat <for [Military] units>"]
```

Comment:
Id write +50%, but personal preference I guess

Reply:
So this works too?

Reply:
It works, but users might interpret this that Military units only earn half of normal XP instead of 1.5 times

Reply:
resolved in https://github.com/ravignir/RekMOD/pull/43/commits/a7f15d56c535c9faaec0fedf201e82619fcf33f5